install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -796,6 +796,12 @@ data:
   bgp-announce-pod-cidr: {{ .Values.bgp.announce.podCIDR | quote }}
 {{- end}}
 
+{{- if .Values.bgpControlPlane.enabled }}
+  enable-bgp-control-plane: "true"
+  {{- else }}
```

Are the leading spaces on purpose here?

```suggestion
{{- else }}
```

---

nope, will fix.